pkg/collector/corechecks/servicediscovery/module/impl_linux_test.go

```diff
@@ -772,10 +772,13 @@ func TestDocker(t *testing.T) {
 	url := setupDiscoveryModule(t)
 
 	dir, _ := testutil.CurDir()
-	err := protocolUtils.RunDockerServer(t, "foo-server",
-		dir+"/testdata/docker-compose.yml", []string{},
+	dockerCfg := dockerutils.NewComposeConfig("foo-server",
+		dockerutils.DefaultTimeout,
+		dockerutils.DefaultRetries,
 		regexp.MustCompile("Serving.*"),
-		protocolUtils.DefaultTimeout, 3)
+		nil,
```

Comment:
Is there a way to avoid the magic `nil` here?

Reply:
originally it was `[]string{}` do you think it is better?

Reply:
I'd recommend of have "DefaultEnv" instead of nil or empty string list